packages/contracts/deploy/60_dao-factory.ts

```diff
@@ -1,39 +1,22 @@
-import {HardhatRuntimeEnvironment} from 'hardhat/types';
-import {DeployFunction} from 'hardhat-deploy/types';
+import { HardhatRuntimeEnvironment } from 'hardhat/types';
+import { DeployFunction } from 'hardhat-deploy/types';
 
-import {getContractAddress} from './helpers';
+import { getContractAddress } from './helpers';
 
 const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
-  const {deployments, getNamedAccounts, ethers} = hre;
-  const {deploy} = deployments;
+  const { deployments, getNamedAccounts, ethers } = hre;
+  const { deploy } = deployments;
 
-  const {deployer} = await getNamedAccounts();
+  const { deployer } = await getNamedAccounts();
```

Comment:
Your IDE doesn't seem to use our `.prettierrc` in which we specify `"bracketSpacing": false`.
We use this plugin for VScode: https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode